src/diffcalc_API/routes/hkl.py

```diff
@@ -30,11 +30,21 @@ async def lab_position_from_miller_indices(
     name: str,
     miller_indices: HklModel = Depends(),
     wavelength: float = Query(..., example=1.0),
+    angles: Optional[List[str]] = Query(default=None, example=["mu", "nu", "phi"]),
```

Comment:
I think "axes" might be a better name than "angles", would also fit better with conventions in other DLS software such as Malcolm

Reply:
:+1: